e2e_test/batch/basic/all_any_some.slt.part

```diff
@@ -116,3 +116,15 @@ drop materialized view mv2;
 
 statement ok
 drop table tmp;
+
+statement ok
+create materialized view mv as with cte(v1, v2) as (values ('a1', array[1,2]), ('a2', array[3,4]), ('b', array[5,6])) select v1, 1 = any(v2) as any from cte where v1 like 'a%';
```

Comment:
We have to carefully avoid any `compact` during execution to reproduce this issue.